staging/src/k8s.io/cloud-provider/controllers/node/node_controller_test.go

```diff
@@ -162,7 +166,7 @@ func TestEnsureNodeExistsByProviderID(t *testing.T) {
 	}
 }
 
-func Test_AddCloudNode(t *testing.T) {
+func Test_SyncNode(t *testing.T) {
```

Comment:
@andrewsykim Also changed the unit test signature since we don't have `AddCloudNode()` function anymore